app/util/Layer.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Util class for Cpsi layers
+ */
+Ext.define('CpsiMapview.util.Layer', {
+    requires: [
+        'BasiGX.util.Map'
+    ],
+    statics: {
+        TIME_DEPENDENT_ATTRIBUTE_NAME: 'isTimeDedendent',
+
+        /**
+         * Function to get all time dependent layers in map
+         * @param {ol.collection} collection (optional) an OpenLayers layer collection
+         * @returns {ol.Layer[]} An array of time dependent layers
+         */
+        getTimeDependentLayers: function(collection) {
```

Comment:
It might make sense to add method to the layer util of BasiGX `getLayersBy()` (we already have `getLayerBy`), can you change this probably?

Reply:
Moved to BasiGX, see here https://github.com/terrestris/BasiGX/pull/354